### DISCUSSION TITLE
Alexa Media Player richiede autenticazione.

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#905] [HUB: Home Assistant]
Alexa media player richiede autenticazione

Ciao a tutti, 

ho configurato 2 giorni fa l'integrazione di alexa media player secondo la guida: [](https://indomus.it/guide/integrare-amazon-alexa-come-media-player-su-home-assistant/)

Questa mattina mi richiede l'autenticazione, ma non vedo il codice captcha, ne in chrome, ne in Edge, ne in app Home assistant.
![image](https://user-images.githubusercontent.com/60674280/137858447-e6f27c03-ac54-4b13-a029-085c854588b3.png)

Qualcuno si è già trovato in questa situazione?
Grazie.

**Comment by calcioscacchi** [comment]
non fate domande, non ho risposte... ma dopo alcuni tentativi, il problema non si ripresenta. Anche se sono abbastanza sicuro di non aver fatto nulla di decisivo per risolverlo.

**Reply by calz64** [reply]
Io ho lo stesso problema e non si vede il Captcha, però non si è risolto, i tentativi che fai cosa sono? 
il pulsante "Invia" non è attivo se non c'è il captcha.

**Reply by calz64** [reply]
Ho smanettato senza un senso preciso, mettendo captcha falso, e si è risolto mah??? 

Spero non si tratti di sniffing chiedo lumi agli esperti?

**Reply by calcioscacchi** [reply]
io ho riavviato home assistant e il captcha è comparso. Il mio problema è che anche se lo inserivo, la schermata successiva mi dava un codice OTP da inserire in una schermata alla quale accedere (di solito) da un link che arriva per SMS. A me però quel link non è mai arrivato. Riprovando a caricare la pagina, non mi ha più chiesto nulla.

Adesso il dubbio mi viene pure a me... aspettiamo pareri più illuminati.

**Comment by calz64** [comment]
questa volta è tornato a chiedermi autenticazione ma non riesco a fare più nulla mi da questo errore. Help!!!
![Cattura](https://user-images.githubusercontent.com/84615011/139318867-d5c53db9-c762-4472-b594-851be6e80031.JPG)

**Reply by bimbumbay** [reply]
Reinstalla l'integrazione,  ho avuto lo stesso problema e ho risolto reinstallando l'integrazione

**Comment by calz64** [comment]
Io ho eliminato sia l'integrazione con la tendina dei tre puntini in basso
a destra e poi ho eliminato anche l'applicazione in Hacs; poi ho rifatto i
passaggi di installazione e configurazione, è andata per ora.

Il giorno gio 4 nov 2021 alle ore 18:25 Maurizio Montanaro <
***@***.***> ha scritto:

**Reply by calz64** [reply]
Io ho eliminato sia l'integrazione con la tendina dei tre puntini in basso a destra e poi ho eliminato anche l'applicazione in Hacs; poi ho rifatto i passaggi di installazione e configurazione, è andata per ora.

**Reply by calcioscacchi** [reply]
ciao, mi tieni aggiornato se ti torna a dare problemi?

**Reply by calz64** [reply]
yes.

**Reply by calcioscacchi** [reply]
ma scusami, eliminare l'integrazione cosa comporta? le entità si ripristineranno? Hai dovuto rifare tutta la configurazione ,anche a livello di amazon AWS?

**Reply by calz64** [reply]
I dispositivi li devi ri-configurare:  l'integrazione devi attivarla con aggiungi dispositivo e il dispositivo con il pulsantino che resetta. No a livello AWS non devi modificare null, rimane attiva e invariata.

**Reply by calcioscacchi** [reply]
@calz64 dopo una settimana non ti vedo scrivere nulla. Tutto bene? :)

**Comment by gsarcl366** [comment]
Se permettete dico anch'io la mia: ho disinstallato e reinstallato HACS, l'integrazione alexa media player, sembrava andasse ma dopo due giorni si è ripresentato il problema: codice Captcha che non si vede. Ho seguito la guida indomus impostando alexa con autenticazione a due fattori, mi viene voglia di non attivare l'autenticazione a due fattori.

**Comment by calz64** [comment]
**@calcioscacchi.** per ora tutto bene.
Giusto ieri si è ripresentato la notifica integrazione da riconfigurare :( -  ho cliccato su ri-configura (fortunatamente non è comparsa la schermata con il captcha illeggibile), è comparsa invece la schermata con i campi già pre-compilati, ho fatto invia e tutto è andato bene. 
**@gsarcl366** non saprei se suggerirti di non impostare l'autenticazione a due fattori, credo che il problema sia la generazione della chiave in automatico; DA GUIDA  "utilizzeremo un codice 2FA “applicativo”, ovvero da dare in pasto all’integrazione così da evitare ... l'invio del codice OTP via SMS ogni volta che necessita autenticazione".

**Reply by calcioscacchi** [reply]
ho provato anche io e per ora tiene. Vediamo se è la soluzione definitiva

**Comment by gsarcl366** [comment]
Riconfigurato: per adesso tiene. Come mai il sensore temperatura di un echo plus che ho, a volte viene rilevato, a volte no. E' giusto poi che  lo stato di alcuni echo configurati in home assistant risultino unavailable? Grazie anticipatamente e buona giornata
